src/angularjs/src/app/compare/compare.controller.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * @ngdoc controller
+ * @name pfb.analysis-jobs.detail.controller:CompareController
+ *
+ * @description
+ * Controller for showing details about an analysis job
+ *
+ */
+(function() {
+    'use strict';
+
+    /** @ngInject */
+    function CompareController($stateParams, Neighborhood, AnalysisJob, $log, $location) {
+        var ctl = this;
+
+        initialize();
+
+        function initialize() {
+            ctl.places = [null, null, null];
+            ctl.getPlace = getPlace;
+            ctl.clearSelection = clearSelection;
+
+            getPlace(0, $stateParams.place1);
+            getPlace(1, $stateParams.place2);
+            getPlace(2, $stateParams.place3);
+        }
+
+        function getPlace(num, uuid) {
+            if (!uuid) {
+                return;
+            }
+
+            ctl.places[num] = {};
+
+            Neighborhood.query({uuid: uuid}).$promise.then(function(data) {
+                ctl.places[num].neighborhood = new Neighborhood(data);
```

Comment:
I agree it looks odd to see the columns load separately, split seconds apart. Then again, it seems to generally be preferred web design to present content sooner, piecemeal, than to leave the user looking at an empty page. @CloudNiner maybe have thoughts on how best to load here?